modules/distribution/src/main/scripts/impl/init.sh

```diff
@@ -27,4 +27,13 @@ script=$( basename "$SOURCE" )
 
 . "$impl"/config.sh
 
+if [[ -z $HADOOP_PREFIX ]]; then
+  echo "HADOOP_PREFIX needs to be set!"
+  exit 1
+fi
+
+$HADOOP_PREFIX/bin/hdfs dfs -mkdir -p /fluo/lib
+$HADOOP_PREFIX/bin/hdfs dfs -copyFromLocal $FLUO_HOME/lib/fluo-api-*.jar /fluo/lib/
```

Comment:
Over time this could be problematic.  Could end up with multiple fluo jars copied into this dir.